lms/static/js/components/tabbed/views/tabbed_view.js

```diff
@@ -31,6 +52,10 @@
                    initialize: function (options) {
                        this.router = options.router || null;
                        this.tabs = options.tabs;
+                       // Convert each view into a TabPanelView
+                       _.each(this.tabs, function (tabInfo) {
+                           tabInfo.tabPanelView = new TabPanelView({tabInfo: tabInfo});
```

Comment:
It might be nice from an encapsulation perspective to just replace the `view` property of `tabInfo` with the new `TabPanelView`. That way, the `TabbedView` can't get at the underlying view -- it needs to go through the `TabPanelView` to render it. If you like this idea it will probably require some `_.clone`-ing to avoid shenanigans with the circular reference between `tabInfo` and its `tabPanelView`.